.github/pull_request_template.md

```diff
@@ -0,0 +1,40 @@
+
+**Checklist:**
+- [ ] You have [signed off your commits](https://www.kubeflow.org/docs/about/contributing/#sign-off-your-commits)
+- [ ] Ensure you follow best practices from our guide. [Contributing](https://github.com/kubeflow/website/blob/master/content/en/docs/about/contributing.md). 
+
+
+**Description of your changes:**
+
+
+### Issue
+
+<!--
+ If this pull request resolves an open issue, please link the issue in the PR
+ description so it will automatically close when the PR is merged.
+
+ See the GitHub documentation for more details and other options:
+
+ https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword
+-->
+
+Closes: #
+
+<!--Additional Information:-->
+### Labels
+<!-- Please include labels below by uncommenting them to help us better review PRs -->
+
+<!-- /area central-dashboard -->
+<!-- /area katib -->
+<!-- /area kserve -->
+<!-- /area model-registry -->
+<!-- /area notebooks -->
+<!-- /area pipelines -->
+<!-- /area spark-operator -->
+<!-- /area trainer -->
+<!-- /area gsoc -->
+<!-- /area website -->
+<!-- /area community -->
+<!-- /area other -->
```

Comment:
Let's remove this one, because we did not create this label.

Also, having an "other" just encourages people to use it, it's fine if no label implies other.